src/api/v1/db/oracledb/doughs-dao.js

```diff
@@ -31,14 +31,6 @@ const doughColumnNames = {
   specialInstructions: 'SPECIAL_INSTRUCTIONS',
 };
 
-/**
- * convert an output bind param name like `nameOut` to a
- * property name like `name` by removing 'Out' at the end of the string
- * @param {string} outBindParamName
- * @returns {string} the property name
- */
-const outBindParamToPropertyName = (outBindParamName) => outBindParamName.slice(0, -3);
-
```

Comment:
some minor refactoring here and in `ingredients-dao.js` to remove code that was duplicated between the three DAO modules